src/main/java/org/jenkinsci/plugins/mesos/MesosComputer.java

```diff
@@ -3,35 +3,42 @@
 import hudson.model.Executor;
 import hudson.model.Queue;
 import hudson.slaves.AbstractCloudComputer;
-import org.apache.commons.lang.NotImplementedException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 /** The running state of a {@link hudson.model.Node} or rather {@link MesosSlave} in our case. */
 public class MesosComputer extends AbstractCloudComputer<MesosSlave> {
 
+  private static final Logger LOGGER = Logger.getLogger(MesosComputer.class.getName());
```

Comment:
The slf4j interface is a little nicer as it allows for string interpolation without evaluation.

```suggestion
  private static final Logger logger = LoggerFactory.getLogger(MesosComputer.class);
```